wtxmgr/tx.go

```diff
@@ -355,11 +359,30 @@ func (s *Store) deleteUnminedTx(ns walletdb.ReadWriteBucket, rec *TxRecord) erro
 // InsertTx records a transaction as belonging to a wallet's transaction
 // history.  If block is nil, the transaction is considered unspent, and the
 // transaction's index must be unset.
-func (s *Store) InsertTx(ns walletdb.ReadWriteBucket, rec *TxRecord, block *BlockMeta) error {
+func (s *Store) InsertTx(ns walletdb.ReadWriteBucket, rec *TxRecord,
+	block *BlockMeta) error {
+	_, err := s.InsertTxCheckIfExists(ns, rec, block)
+	return err
+}
+
+// InsertTxCheckIfExists records a transaction as belonging to a wallet's
```

Comment:
Not the most scrutable name, can't think of any other good alternatives, so it should do...